opal/mca/hwloc/base/hwloc_base_util.c

```diff
@@ -310,13 +310,29 @@ int opal_hwloc_base_get_topology(void)
             free(shmemfile);
             if (0 != hwloc_shmem_topology_adopt(&opal_hwloc_topology, fd,
                                                 0, (void*)addr, size, 0)) {
-                OPAL_ERROR_LOG(OPAL_ERR_FILE_READ_FAILURE);
-                return OPAL_ERR_FILE_READ_FAILURE;
+                if (4 < opal_output_get_verbosity(opal_hwloc_base_framework.framework_output)) {
+                    FILE *file = fopen("/proc/self/maps", "r");
+                    if (file) {
+                        char line[256];
+                        opal_output_verbose(5, opal_hwloc_base_framework.framework_output,
```

Comment:
Don't use opal_output_verbose here as you already checked the verbosity - just use opal_output(0,...